build/yaml/crd/vpc/crd.nsx.vmware.com_subnets.yaml

```diff
@@ -176,7 +201,7 @@ spec:
                 items:
                   type: string
                 type: array
-              shared:
+              preCreated:
```

Comment:
is this an intended change?

Reply:
FYI @jianjuns ， WDYT？

Reply:
I would not use "preCreated" in SV API. In the VCF-A context, "shared" makes more sense; or if you have a better name to express this Subnet is created outside the Namespace?

Reply:
I do not have a good name... I changed back to "shared" for now.